tools/Dashboard/DevHome.Dashboard/ComSafeWidgetObjects/ComSafeWidget.cs

```diff
@@ -328,26 +339,21 @@ private async Task LazilyLoadOopWidget()
     }
 
     /// <summary>
-    /// Get a widget's ID from a widget object. Tries multiple times in case of COM exceptions.
+    /// Get a widget's ID from a widget object.
     /// </summary>
     /// <param name="widget">Widget</param>
     /// <returns>The Widget's Id, or in the case of failure string.Empty</returns>
     public static async Task<string> GetIdFromUnsafeWidgetAsync(Widget widget)
     {
-        var retries = 5;
-
         return await Task.Run(() =>
         {
-            while (retries-- > 0)
+            try
             {
-                try
-                {
-                    return widget.Id;
-                }
-                catch (Exception ex)
-                {
-                    Log.Warning(ex, $"Failed to operate on out-of-proc object with error code: 0x{ex.HResult:x}, try {retries} more times");
-                }
+                return widget.Id;
+            }
+            catch (Exception ex)
+            {
+                Log.Warning(ex, $"Failed to operate on out-of-proc object with error code: 0x{ex.HResult:x}");

```

Comment:
Is this exception coming from COM, or the OOP object?  If from the object, any exception is returned as a Generic Exception and the message will also be "Generic Exception"

Reply:
I'm specifically worried about COM, I don't _think_ we're expecting we can get any other exception here from the object.